src/pages/Home.jsx

```diff
@@ -4,8 +4,11 @@ import HackDescription from '../components/Home/HackDescription';
 import PhotoCarousel from '../components/Home/PhotoCarousel';
 import FAQSection from '../components/Home/FAQSection';
 import Announcement from '../components/Home/Announcement';
+import useTitle from '../components/General/useTitle';
 
 export default function Home() {
+	useTitle('Home');
```

Comment:
For home page, we can leave header as just `UCLA ACM Hack`. (No need to add `| Home` afterwards)